test/windows/acceptance/logstash_release_default_plugins.ps1

```diff
@@ -0,0 +1,30 @@
+# Created By: Gabriel Moskovicz
+#
+# To be run on Jenkins
+#
+# Requirements to run the test:
+# 
+# - Powershell 4
+# - Windows 7 or newer
+# - Java 7 or newer
+# - Ruby 7 or newer
+
+$ruby = $env:RUBY_HOME  + "\jruby.exe"
+
+$env:SPEC_OPTS="--format documentation"
```

Comment:
Sounds good. i think that this was inherited from the test that we did a while ago to understand what was failing.